### PR TITLE
i18n: translations

### PR DESCRIPTION
**Pull request by umlaeute**
this PR is for keeping translations on `master` uptodate.
no functionality is added, just the translation files for the interface.
as such it should be always safe to merge this into `master`.

while i've updated the help for German for the upcoming 0.53 release, help is needed from other native speakers (@Lucarda , @porres , @Ant1r , @mxa , and whoever feels like it).

it would be great if we could find a more automatic workflow...

### CODE REVIEW

**Comment by porres** [comment]
I tried creating a PR and it failed, can you help me with that?

**Comment by umlaeute** [comment]
Weird. It shouldn't be any different from creating any other PR. If you post a link to your branch, I could have a look

**Comment by Ant1r** [comment]
What "failed"? #1774

**Comment by mxa** [comment]
Should we perhaps use https://www.transifex.com/open-source/ ?

**Comment by eighthave** [comment]
Hey all!  At this point, I would recommend using https://hosted.weblate.org. 
Its free software!

File: po/es.po [hunk 1]
```diff
@@ -311,9 +309,7 @@ msgid "Pd window"
 msgstr "Ventana de Pd"
 
 msgid "Match whole word only"
-msgstr ""
-"Comparar solo con\n"
```
Comment (Lucarda) [inline po/es.po:365]: this `\n` was intentional as it fits better in the button.

I'll try to reintroduce it via `weblate.org`

Comment (umlaeute) [reply po/es.po:365]: i thought so, but didn't really check.

maybe you can add a note on weblate

Comment (Lucarda) [reply po/es.po:365]: done

**Pull request merged**